real_time/arduino_src/lib_avr/radioreceiver/RadioReceiver.cpp

```diff
@@ -1,48 +1,31 @@
-#include "servoreceiver.h"
+#include "RadioReceiver.h"
 
 
-inline long map(long x,
-                long in_offset,
-                long in_scale,
-                long out_offset,
-                long out_scale) 
-{
-    return ((x - in_offset) * out_scale / in_scale) + out_offset;
+RadioReceiver::RadioReceiver() {
```

Comment:
The style that's currently in `main.cpp` has the curly brace on the next line from function name. Are we looking to change this or was this just a simple mistake?